playwright/accessibility/SheetMobile.spec.mjs

```diff
@@ -0,0 +1,8 @@
+// @flow strict
```

Comment:
This file will need to be renamed to match the new format

Reply:
DONE